<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClusterEditor</class>
 <widget class="QWidget" name="ClusterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cluster Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QListView" name="listViewClusterList">
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonRemoveCluster">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="list-remove">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAddCluster">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="list-add">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelServerUrl">
         <property name="text">
          <string>Server url:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditServerUrl"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelTlsName">
         <property name="text">
          <string>TLS Server name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTlsName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="labelProxyUrl">
         <property name="text">
          <string>Proxy URL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditProxyUrl"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelCaData">
         <property name="text">
          <string>Certificate authority data (CA):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCaData"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="labelCaFile">
         <property name="text">
          <string>Certificate authority file (CA):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCaFile"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCaFileSearch">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="edit-find">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBoxInsecureSkipTlsVerify">
         <property name="text">
          <string>Allow INSECURE skip TLS verification</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxDisableCompression">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Disable compression</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="pushButtonRevertCluster">
         <property name="text">
          <string>Revert</string>
         </property>
         <property name="icon">
          <iconset theme="document-revert">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCloneCluster">
         <property name="text">
          <string>Clone cluster</string>
         </property>
         <property name="icon">
          <iconset theme="edit-copy">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSaveCluster">
         <property name="text">
          <string>Save cluster</string>
         </property>
         <property name="icon">
          <iconset theme="document-save">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
